dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -1136,7 +1138,20 @@ private String formatObject(Object objs, int depth) {
             if (Proxy.isProxyClass(cls) || className.contains("$Proxy$_$$_Weld")) {
                 return "Proxy for " + className + "@" + Integer.toHexString(System.identityHashCode(objs));
             }
-            return objs.toString();
+//            return objs.toString();
+            String s = null;
+            try {
+                s = AccessController.doPrivileged(new PrivilegedExceptionAction<String>() {
+                    @Override
+                    public String run() throws Exception {
+                        return objs.toString();
+                    }
+                });
+
+            } catch (PrivilegedActionException pae) {
+                pae.printStackTrace();
+            }
+            return s;
```

Comment:
@fmhwong and @Channyboy this is the change we would like your review on. As a bit of background, we’ve been working on the new JAX-WS feature for Jakarta EE, and the implementation we use has changed they way they log request/responses. This PR is the switch to their new Logging feature, and when initializing it caused some Java 2 Sec errors to be thrown. I suggested @BerksanAtes add a doPriv to spot it was occurring in the logging code. To be clear: in the original Java 2 sec failure there was no JAX-WS code in the stack, so no where that we could add a doPriv to our own code.

Reply:
I remember catching exceptions with stack trace trough PrivilegedActionException in our bundles. Having said that, It might be returning different results on your code base. Do you want me to remove it?

Reply:
Yes.  The `catch (Exception e)` block should catch the `PrivilegedActionException` so we have a consistent error message.

Reply:
I see now. Thank you! I changed it.